### ISSUE TITLE
Could you please provide a requirements.txt or readme.md?

### ISSUE THREAD

**Issue by TianHongZXY** [shalini1194/SAKT#1]
I'm recently tyring to improve my dkt model performance on my datasets，I found your paper helpful to me，but the code without a readme.md and requirements.txt is a little bit hard to read，so could you please offer some information that can help me understand the code faster? Thank you!

**Comment by saxenashreyaa** [comment]
> I'm recently tyring to improve my dkt model performance on my datasets，I found your paper helpful to me，but the code without a readme.md and requirements.txt is a little bit hard to read，so could you please offer some information that can help me understand the code faster? Thank you!

Yes please provide the requirements.txt, that would help me a lot
